bigtable-client-core-parent/bigtable-client-core/pom.xml

```diff
@@ -133,7 +133,6 @@ limitations under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-release-plugin</artifactId>
-                <version>2.5.1</version>
```

Comment:
How come the version was removed in this case? I guess that makes it default to 'latest' but still makes me a bit nervous, maybe we can pin it more explicitly?